### ISSUE TITLE
Updating UI/UX of the website

### ISSUE THREAD

**Issue by aryanprince** [aryankashyap7/CheatSheets-for-Developers#218]
## Design Ideas
- Fix casing for the buttons on homepage since all of them now just use Title Case
- Add small logos of the language/framework to the left of the buttons

| Before| After | 
| ------------- |:-------------:| 
| <img width="400" src="https://user-images.githubusercontent.com/45622345/199053634-d5f587fe-c6a7-487c-8f1f-3db44698293d.png">     | <img width="400" src="https://user-images.githubusercontent.com/45622345/199053475-8c32d855-d965-4e4c-a022-9059eb2e9696.png"> | 

### Some minor changes
- Add 2 columns on larger viewports so you don't have to scroll as much
- Make the description sentence case

| Before| After | 
| ------------- |:-------------:| 
| ![image](https://user-images.githubusercontent.com/45622345/199054956-d8adde5e-0cbf-4f09-997a-2a9101778541.png)     | ![image](https://user-images.githubusercontent.com/45622345/199054966-3893978f-7a18-4a3b-bb88-7990b502a494.png) | 

@aryankashyap7 @Dhravya Thoughs?

**Comment by Dhravya** [comment]
Yeah. good ideas actually. We could do it

It would be hard to maintain the logos though. How would that work? https://devicon.dev/ has a lot of them, but for things like Competitive programming, where there's no particular icon, what do we do?

BTW, the longer viewports is already done (It's 3 column now, and decreases with device)

**Comment by aryanprince** [comment]
Yeah, I was hoping we could use [https://devicon.dev/](https://devicon.dev/). 

Not sure about the competitive programming ones, maybe we use the language logo + find a generic logo from [https://icons8.com/](https://icons8.com/) or something to indicate it's for 'competitive programming'.

@Dhravya And what about the casing of the buttons? I think rn you just take the names of the .md files and just make it title case yeah?

**Comment by aryanprince** [comment]
@nexxeln Thoughts?

**Comment by aryanprince** [comment]
@Dhravya A few more bugs showing up now

- Weird spacing for buttons
- Tiny purple lines visible on some buttons
![image](https://user-images.githubusercontent.com/45622345/199457463-863573ca-6434-40f5-9dde-e0f519aac8e6.png)

**Comment by Dhravya** [comment]
Yes that's something to fix

also maybe the text inside buttons should be justified to center

would you like to take this issue?

**Comment by aryanprince** [comment]
Yeah, I'll have a look
